packages/jest-core/src/TestScheduler.ts

```diff
@@ -424,11 +402,11 @@ class TestScheduler {
    * to make dealing with them less painful.
    */
   private _getReporterProps(reporter: string | Config.ReporterConfig): {
-    path: string;
     options: Record<string, unknown>;
+    path: string;
   } {
     if (typeof reporter === 'string') {
-      return {options: this._options, path: reporter};
+      return {options: {}, path: reporter};
     } else if (Array.isArray(reporter)) {
       const [path, options] = reporter;
```

Comment:
add a check that tuple length is 2?

Reply:
Can we leave this for another PR? Currently this method is called twice, that's nothing significant, but still unnecessary. I have an idea how to rework it, but that's another story (;